### ISSUE TITLE
can we stylise the way a quote is displayed?

### ISSUE THREAD

**Issue by gvklooster** [redhousestudio/redhousestudio.github.io#15]
on projects/joep-beving-prehension/

![screen shot 2018-01-05 at 12 01 49](https://user-images.githubusercontent.com/1320087/34606702-b12358fa-f210-11e7-80c8-d916b512345c.png)

I would like it to be same font (size) as normal body text, but in a slightly lighter color and italic

**Comment by faizanvahevaria** [comment]
@GFunkloostar 
Yesterday, when we were on the call, you said that this is not in the version 1.0

**Comment by gvklooster** [comment]
true, but this just looks a bit silly! doesn't it.. ;) I think silly should not be part of a good website.
